root/inc/activity.tx

```diff
@@ -1,4 +1,4 @@
 <div class="activity-graph">
-    <img src="[% $url %]?res=month" />
+    <img src="[% $url %]?res=month" alt="Activity Graph"/>
```

Comment:
This is a bad alt text. It just repeats the heading directly above it without replacing the information presented in the graph.

One option would be to count the number of groups in the graph and describe how high each of them are. 

The other choice is to alt it as "Changes" and link to the whole thing to the `Changes` file